ui/src/main/js/global/GlobalTestModal.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Modal } from 'react-bootstrap';
+
+const GlobalTestModal = ({
+    children, showTestModal, handleTest, handleCancel
+}) => (
+    <Modal show={showTestModal} onHide={handleCancel}>
+        <Modal.Header closeButton>
+            <Modal.Title>Test Your Configuration</Modal.Title>
```

Comment:
It might be nice to have a default prop for this.

Reply:
I'd rather not have these as props until we actually use the prop functionality. Trying to stick to as static as possible

Reply:
Static doesn't mean "not-reusable" it just means we won't be generating this based on a request-response with the Alert server. So like with "Send Test Message" below, that is still unique to email. Testing Global Jira is totally different.

Reply:
What is the message on the globa jira test? I don't see it use a global test modal at all

Reply:
My mistake, Jira doesn't use a modal, but it does a different _type_ of test than email. I still think the point about reusability is different than being dynamically generated though.

Reply:
I agree that we want to make it reusable which is why I created the new component as a separate item instead of built into the GlobalChannel component. Using as few props as possible makes it easier to read and understand in my opinion as you'll have to check multiple places to figure out what the resulting value is. If we end up needing to change any hardcoded value it's very simple to add a prop as you suggested but until then I say we keep the content as simple as possible.